Runtime/ModularAvatarGlobalCollider.cs

```diff
@@ -0,0 +1,58 @@
+﻿#region
+
+using UnityEngine;
+
+#endregion
+
+namespace nadena.dev.modular_avatar.core
+{
+	//All available VRChat Colliders
+	public enum GlobalCollider
+	{
+		Head,
+		Torso,
+		HandLeft, HandRight,
+		FingerIndexLeft, FingerIndexRight,
+		FingerMiddleLeft, FingerMiddleRight,
+		FingerRingLeft, FingerRingRight,
+		FingerLittleLeft, FingerLittleRight,
+		FootLeft, FootRight,
+		None,
+	}
+
+	[AddComponentMenu("Modular Avatar/MA Global Collider")]
+	[HelpURL("https://modular-avatar.nadena.dev/docs/reference/global-collider?lang=auto")]
+	public class ModularAvatarGlobalCollider : AvatarTagComponent
+	{
+		//Toggle for manual remapping, if false auto remapping.
+		public bool manualRemap = false;
+
+		//descriptor collider to replace (Only used in manual mode)
+		public GlobalCollider sourceCollider;
+
+		//May be better to do this with the method bone proxy uses
+		public AvatarObjectReference rootTransform = new AvatarObjectReference();
+		public GameObject remapTargetObject => rootTransform.Get(this);
+
+		public bool copyOriginalShape = false;
+		public bool visualizeGizmo = true;
+		public float radius = 0.05f;
+		public float height = 0.2f;
+		public Vector3 position = Vector3.zero;
+		public Quaternion rotation = Quaternion.identity;
```

Comment:
Somewhere we should document the axis the capsule's height corresponds to (is it Y or Z?)

Reply:
Code and doc page probably